docs/vtex-io/Storefront-Guides/styling-your-store/vtex-io-documentation-using-css-handles-for-store-customization.md

```diff
@@ -1,50 +1,52 @@
 ---
-title: "Using CSS Handles for store customization"
+title: "Using CSS handles for store customizations"
 slug: "vtex-io-documentation-using-css-handles-for-store-customization"
 hidden: false
 createdAt: "2020-06-03T16:02:45.024Z"
 updatedAt: "2022-12-13T20:17:44.398Z"
 ---
 
+## Introduction
 
-This guide will walk you through how to use CSS Handles to customize your storefront. CSS Handles are unique identifiers assigned to HTML elements. They can be used in the store theme to customize frontend components via CSS classes.
+CSS handles are **HTML element identifiers**. Therefore, they can be used to customize any store component by simply using CSS classes through theme code of the store.
 
+CSS handles are basically the **layout building assistants** for your store. Follow the steps below and learn how to use them in a fast and simple way!
 
-## Step by step
+## Instructions
 
-1. Open your workspace on your browser, add `?__inspect` at the end of the page URL, and press enter. For example, `https://{workspace}--{account}.myvtex.com?__inspect`. Please note that it must be a development workspace rather than a production one, and it must be under the domain `myvtex.com`.
+1. While viewing your workspace on your browser, add `?__inspect` at the end of the page URL and press enter. For example, `https://yourworkspace--youraccount.myvtex.com?__inspect`. Note that the workspace must be a development workspace rather than a production one, and it must be under the domain `myvtex.com`.
```

Comment:
```suggestion
1. Access the following URL: `https://{workspace}--{account}.myvtex.com?__inspect`, replacing the values between curly braces according to your scenario. Please note that you must use a development workspace rather than a production one, and the store domain must be under `myvtex.com`.
```